fastestimator/dataset/data/tednmt.py

```diff
@@ -26,7 +26,7 @@
 
 
 def _read_data(file_path: str) -> List[str]:
-    with open(file_path) as f:
+    with open(file_path, encoding='utf-8') as f:
```

Comment:
Running the notebook failed in windows since the default is different. So setting encoding='utf-8' to avoid this error in future.